test/unittests/stack_test.cpp

```diff
@@ -281,7 +281,10 @@ TEST(operand_stack, to_vector)
     stack.push(2);
     stack.push(3);
 
-    EXPECT_THAT(std::vector(stack.rbegin(), stack.rend()), ElementsAre(1, 2, 3));
```

Comment:
Tried `ElementsAre(fizzy::Value{1}, ...)` but that complained there's no `operator=(const Value, const Value)` and it had to be fully qualified with `fizzy::` because otherwise conflicted with `Value` in gmock/gtest -- all in all, this is simpler.

Reply:
Could also add check for size of `result` to equal 3 here.

Reply:
Done.